### PR TITLE
logical_select: add a test fot load_table option

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: test/command/suite/sharding/logical_select/load_table/load_table.test [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+]
+load --table Logs_20150204
+[
+{
+  "timestamp": "2015-02-04 00:00:00",
+  "memo":      "2015-02-04 00:00:00"
+}
+]
+logical_select --logical_table Logs --shard_key timestamp --load_table Temp --load_columns "memo, timestamp" --load_values "timestamp, memo"
```
Comment (kou) [inline test/command/suite/sharding/logical_select/load_table/load_table.test]: It should be broken to multiple lines.

Comment (kou) [reply test/command/suite/sharding/logical_select/load_table/load_table.test]: It should use different column names.

**Comment by komainu8** [comment]
@kou 
Thank you for your comments.
I've fixed them.

**Comment by komainu8** [comment]
This pull request close. Because It has already been implemented.

**Pull request closed without merging**